### PR TITLE
404-page added

### PR DESCRIPTION
**Pull request by surbhi594**
I have fixed the issue-166

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -126,7 +126,9 @@
             <a class="entypo-linkedin"><span>LinkedIn</span></a>
         </li>
         <li>
-            <a class="entypo-github"><span>Github</span></a>
+
+          <a href="https://github.com/Algo-Phantoms/Algo-Phantoms-Frontend" target="blank" class="entypo-github"><span>Github</span></a>
```
Comment (harikesh409) [inline index.html:130]: The `target` attribute value is wrong it should be `_blank`.

File: 404_page/404_p.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            <div class="line"></div>
+            <h1 id="word" class="glitch_word0">Oops!Page not found</h1>
+            <h1 id="word1" class="glitch_word1">Oops!Page not found</h1>
+            <h1 id="word2" class="glitch_word2">Oops!Page not found</h1>
+          </div>
+
+         <div>
+             <img src="404_logo.png" alt="logo">
+          <h4>We can't find the page you're looking for.</h4>
+          <a style="text-decoration: none;" class="main-btn" href="https://algo-phantoms.github.io/Algo-Phantoms-Frontend/#" role="button">GO BACK HOME</a>
```
Comment (harikesh409) [inline 404_page/404_p.html:33]: In `href` instead of giving a static home page url point to `index.html`

**Review by harikesh409 — CHANGES_REQUESTED** [review]
Github 404 pages should be present in the root folder itself.

Ref: https://help.github.com/articles/custom-404-pages

**Comment by harikesh409** [comment]
@surbhi594 are you working on the changes?

**Comment by surbhi594** [comment]
Yes sir. It will be done by tomorrow

**Comment by harikesh409** [comment]
Closing it as a new PR #232 is raised.

**Pull request closed without merging**